[Blog] Kubecon EU 2023 (#2534)
open-telemetry/opentelemetry.io: content/en/blog/2023/kubecon-eu.md

  by Pavol Loffay, Red Hat & Jonah Kowall, Aiven<br> Wednesday, April 19 •
  11:55 - 12:30
- **[Using OpenTelemetry for Application Security, with a Real Life Example](https://sched.co/1HyYT)**<br>
  by Ron Vider, Oxeye<br> Wednesday, April 19 • 11:55 - 12:30
- **[Understand Systems with OpenTelemetry: A Hybrid Telemetry Data Backend](https://sched.co/1Hyb2)**<br>
  by Ran Xu, Huawei & Xiaochun Yang, Northeastern University<br> Wednesday,
  April 19 • 14:30 - 15:05
- **[OTel Me About Metrics: A Metrics 101 Crash Course](https://sched.co/1Hya1)**<br>
  by Reese Lee, New Relic<br> Wednesday, April 19 • 17:25 - 18:00
- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05
- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
  17:30

> ## Co-located Events

Reviewer: There will be as well Jaeger project booth, do you want to list it here?

Author: Let me think about that, because if we list jaeger, we might also consider to list other observability projects (prometheus, pixie, ...)

Author: not really sure how to incorporate that best, if you have any suggestions let me know, otherwise I will skip it.

The [OpenTelemetry project meeting](https://sched.co/1JWS7) is to meet and
network with the OpenTelemetry maintainers and core contributors. This meeting
will be held 13:00 - 15:00 on Tuesday April 18, 2023.

[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
of cloud-native observability projects_. This event will be held on April 18,

> 2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.

Reviewer 2: I think it would be worth pointing out that for the project meeting, one does _not_ need an all-access pass (right?) but for the Observability Day they would and those are already sold out as far as I know.
The ticket structure was recently changed which has led to some confusion already.

Reviewer 2: ```suggestion
The [OpenTelemetry project meeting](https://sched.co/1JWS7) is to meet and
network with the OpenTelemetry maintainers and core contributors. This meeting
will be held 13:00 - 15:00 on Tuesday April 18, 2023.
_This meeting can be attended with the standard in-person pass and does not require an "all-access" pass._

[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
of cloud-native observability projects_. This event will be held on April 18,
2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.
_This co-located event is not included in the general in-person pass but only in the "in-person all-access" pass. See the [Kubecon website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/) for more details._
```
^ at least that's my understanding

Author: You're right @arminru! That's a good call out

## OpenTelemetry Project Booth

Drop by and say Hi! at the OpenTelemetry project booth in the Kubecon EU Project

> Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag!

Reviewer 3: oh nice :D

Reviewer 4: @svrnm Thanks for putting this post together. Can you add an image or two from Kubecon NA as well as for OTel swag :-)

Author: @alolita happy to do so, but I don't have any images, so if you have some, please send them over and I can add them in :-)

Author: I'll leave it like it is for now, if anyone has something to add, we can do this through a follow up PR

- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05
- **[Connected Observability Pipelines in the OpenTelemetry Collector](https://sched.co/1HyXb)**<br>
  by Daniel Jaglowski, observIQ<br> Friday, April 21 • 11:55 - 12:30
- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
  17:30

## Co-located Events

Come network with OpenTelemetry maintainers and core contributors during the
[OpenTelemetry project meeting](https://sched.co/1JWS7), on Tuesday April 18,
2023 from 13:00 - 15:00. You can attend with a _standard in-person pass_.

[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
of cloud-native observability projects_. This event will be held on April 18,
2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.

> > <i class="far fa-exclamation-triangle"></i> **IMPORTANT access note**: You

Reviewer 5: Please add that observability day will also be live-streamed if you have a virtual ticket

Author: fixed

OpenTelemetry or contributing to the project.

Below you will find a list of talks and other activities related to
OpenTelemetry.

## KubeCon Talks and Maintainer Sessions

- **[Jaeger: The Future with OpenTelemetry and Metrics](https://sched.co/1HySf)**<br>
  by Pavol Loffay, Red Hat & Jonah Kowall, Aiven<br> Wednesday, April 19 •
  11:55 - 12:30
- **[Using OpenTelemetry for Application Security, with a Real Life Example](https://sched.co/1HyYT)**<br>
  by Ron Vider, Oxeye<br> Wednesday, April 19 • 11:55 - 12:30
- **[Understand Systems with OpenTelemetry: A Hybrid Telemetry Data Backend](https://sched.co/1Hyb2)**<br>
  by Ran Xu, Huawei & Xiaochun Yang, Northeastern University<br> Wednesday,
  April 19 • 14:30 - 15:05
- **[OTel Me About Metrics: A Metrics 101 Crash Course](https://sched.co/1Hya1)**<br>
  by Reese Lee, New Relic<br> Wednesday, April 19 • 17:25 - 18:00
- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05

> - **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
>   by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
>   Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
>   17:30

Reviewer 6: ```suggestion
- **[Connected Observability Pipelines in the OpenTelemetry Collector](https://sched.co/1HyXb)**<br>
  by Daniel Jaglowski, observIQ<br> Friday, April 21 • 11:55 - 12:30
- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
  17:30
```

Reviewer 2: Nice! 👏

  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
  17:30

## Co-located Events

Come network with OpenTelemetry maintainers and core contributors during 
the [OpenTelemetry project meeting](https://sched.co/1JWS7), on Tuesday April 18, 2023
from 13:00 - 15:00. You can attend with a _standard in-person pass_.

[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
of cloud-native observability projects_. This event will be held on April 18,
2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.

> _This co-located event is not included in the general in-person pass but only in
> the "in-person all-access" pass. See the
> [KubeCon website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/)
> for more details._

Reviewer 7: ```suggestion

> <i class="far fa-exclamation-triangle"></i> **IMPORTANT access note**: You
> need an _in-person all-access_ pass for access to **Observability Day**. For
> details, see [KubeCon
> registration](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/).
```

## OpenTelemetry Project Booth

Drop by and say Hi! at the OpenTelemetry project booth in the KubeCon EU Project
Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag!

> You will find us in the Solutions Showcase in Hall 5, Kiosk Number 20.
> 
> [The pavilion hours are][]:

Reviewer 7: ```suggestion
You will find us in the Solutions Showcase in Hall 5, Kiosk Number 20. [Pavilion hours][] are as follows:
```

You'll also need the corresponding change to the mardown-link definition made below.

- Wednesday, April 19: 10:30 – 21:00 (includes KubeCrawl + CloudNativeFest from
  18:00 – 21:00)
- Thursday, April 20: 10:30 – 17:30
- Friday, April 21: 10:30 – 14:30

We’re super excited to learn from all of you about your experience with
OpenTelemetry, how you are using OpenTelemetry, features you may need and issues
you may be running into when using OpenTelemetry.

> Come join us to listen, learn and get involved in OpenTelemetry.

Reviewer 7: ```suggestion
Come join us to listen, learn, and get involved in OpenTelemetry.
```

---
title: Join us for OpenTelemetry Talks and Activities at KubeCon EU 2023
linkTitle: KubeCon EU '23
date: 2023-03-30
spelling:
  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun Alolita
  Sharma Dyla Benedikt Bongartz Oliveira Pathak
author: '[Severin Neumann](https://github.com/svrnm)'
---

> The OpenTelemetry project maintainers, and members of the governance committee
> and technical committee are excited to be at [KubeCon EU][] in Amsterdam from
> April 18 - 21, 2023!

Reviewer 7: ```suggestion
The OpenTelemetry project maintainers, members of the governance committee,
and technical committee are excited to be at [KubeCon EU][] in Amsterdam from
April 18 - 21, 2023!
```

> You can meet us at the [project booth][], listen to our talks or learn with us

Reviewer 8: As an end-user reading this, I immediately wonder: when and where can I meet folks at the booth? Is that every day all day? Where?

Author: I changed the intro to _Read on to learn about all the things related OpenTelemetry during KubeCon._ and leave it to the rest of the content to guide people to the right dates and locations, wdyt?

> You can meet us at the [project booth][], listen to our talks or learn with us
> during workshops. We are looking forward to hear from you, how you are using
> OpenTelemetry or contributing to the project.

Reviewer 7: ```suggestion
Come meet us at the [project booth][], listen to our talks or learn with us
during workshops. We are looking forward to hear from you, in particular how you are using
OpenTelemetry or contributing to the project.
```

> Below you will find a list of talks and other activities related to
> OpenTelemetry.

Reviewer 7: ```suggestion
You will find below a list of talks and other activities related to
OpenTelemetry.
```

> ## KubeCon Talks and Maintainer Sessions

Reviewer 8: Add EMEA end-user session? CC: @martinjt

Author: yes, please, gimme the times & dates and I add them

Reviewer 8: So, we haven't fixed that yet (unless I'm missing something), would it make sense to do it during the pavilion hours?

Author: I would guess so, probably worth checking at #otel-maintainers slack.
